src/cdk/testing/protractor/protractor-element.ts

```diff
@@ -40,7 +40,8 @@ const keyMap = {
   [TestKey.F10]: Key.F10,
   [TestKey.F11]: Key.F11,
   [TestKey.F12]: Key.F12,
-  [TestKey.META]: Key.META
+  [TestKey.META]: Key.META,
+  [TestKey.SPACE]: Key.SPACE
```

Comment:
I think the reason space was not on this list is because we just used the string `' '`. Is there a reason we can't do that in this PR?

Reply:
I wasn't aware that was supported. Reverting.